LiteFarmOrg/LiteFarm
# F21-627/french my farm icon added

Ticket [F21-627](https://lite-farm.atlassian.net/browse/F21-627)

To Test:
1. Change the language to French
2. Check that My Farm icon is displayed in French ("ma ferme")
